### ISSUE TITLE
conda installation

### ISSUE THREAD

**Issue by ValeryPolyakov** [PatWalters/rd_filters#5]
Hi Pat, Do you support conda installation of rd_filters? Thanks, Valery

**Comment by PatWalters** [comment]
It's not conda installable (e.g. conda install rdfilters) but it can certainly be installed in a conda environment.

**Comment by PatWalters** [comment]
This is not currently supported, I can look into it.

**Comment by ValeryPolyakov** [comment]
Thanks Patrick! This would be very helpful. Valery

**Comment by hsiaoyi0504** [comment]
Actually, under conda environment, I think you can still install it through `pip`.

**Comment by noorikhalifa** [comment]
Can this be installed in a conda environment?? I'm getting an error that says - ModuleNotFoundError: No module named 'rd_filters'

**Comment by PatWalters** [comment]
Yes, it can be installed in a conda environment.  I just tested it 

conda create -c conda-forge -n rdkit_test rdkit
conda activate rdkit_test
pip install git+https://github.com/PatWalters/rd_filters.git
which rd_filters
/opt/anaconda3/envs/rdkit_test/bin/rd_filters
rd_filters -h
Usage:
rd_filters filter --in INPUT_FILE --prefix PREFIX [--rules RULES_FILE_NAME] [--alerts ALERT_FILE_NAME][--np NUM_CORES]
rd_filters template --out TEMPLATE_FILE [--rules RULES_FILE_NAME]

Options:
--in INPUT_FILE input file name
--prefix PREFIX prefix for output file names
--rules RULES_FILE_NAME name of the rules JSON file
--alerts ALERTS_FILE_NAME name of the structural alerts file
--np NUM_CORES the number of cpu cores to use (default is all)
--out TEMPLATE_FILE parameter template file name

You may want to try deactivating and reactivating your conda environment.  I find that sometimes conda doesn't pick up new packages until you reactivate.

**Comment by UnixJunkie** [comment]
this issue could be closed

**Comment by PatWalters** [comment]
done
